tests/c12_config.yml

```diff
@@ -1,7 +1,7 @@
 data_table: default
 diag_table: default
 experiment_name: default
-forcing: "gs://vcm-fv3config/data/base_forcing/v1.1/"
+forcing: gs://vcm-fv3config/data/base_forcing/v1.1/
```

Comment:
Funny, I would've opted to put quotes around these URLs in YAML, mostly because they contain characters with special meaning in YAML such as `:` which can easily lead to unwanted errors. See also [stachoverflow](https://stackoverflow.com/questions/19109912/yaml-do-i-need-quotes-for-strings-in-yaml).